base/applications/notepad/dialog.c

```diff
@@ -81,13 +81,14 @@ void UpdateWindowCaption(BOOL clearModifyAlert)
     /* When a file is being opened or created, there is no need to have the edited flag shown
        when the new or opened file has not been edited yet */
     if (clearModifyAlert)
-        StringCbPrintf(szCaption, ARRAY_SIZE(szCaption), _T("%s - %s"), szFilename, szNotepad);
+        StringCbPrintf(szCaption, sizeof(szCaption), _T("%s - %s"),
```

Comment:
Since you are at it could you please put the contents of this if-block inside curly brackets?

Reply:
Right. Done.